packages/datetime/src/datePickerCaption.tsx

```diff
@@ -101,16 +105,19 @@ export class DatePickerCaption extends React.Component<IDatePickerCaptionProps,
         this.positionArrows();
     }
 
-    private monthArrowRefHandler = (r: HTMLElement) => this.monthArrow = r;
-    private yearArrowRefHandler = (r: HTMLElement) => this.yearArrow = r;
+    private containerRefHandler = (r: HTMLElement) => this.containerElement = r;
+    private monthArrowRefHandler = (r: HTMLElement) => this.monthArrowElement = r;
+    private yearArrowRefHandler = (r: HTMLElement) => this.yearArrowElement = r;
```

Comment:
i'm accumulating a list of major refactors to datetime code, for the new year... will add this!